apps/website/src/content/docs/typography.mdx

```diff
@@ -1,17 +1,56 @@
 ---
 title: Typography
-description:
+description: Typography components provide consistency in text styling across the application.
 thumbnail: Typography
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='text--basic' />
-
 ## Text
 
-<LiveExample src='Text.main.jsx' truncate={false}>
-  <AllExamples.TextMainExample client:load />
+`Text` is a [polymorphic](#polymorphic-only-semantics-and-not-size) typography component that can be used to render any kind of text as any kind of element.
+
+```jsx
+<Text>This is a text.</Text>
+```
+
+### Variant (Only size and not semantics)
+
+The `variant` prop can be used to determine which type of typography size should be utilized for the content of the application. The available variants are:
+
+- `"headline"`: `<h1>` sized text.
+- `"title"`: `<h2>` sized text.
+- `"subheading"`: `<h3>` sized text.
+- `"leading"`: `<h4>` sized text.
+- `"body"` (default): `<p>` sized text.
+- `"smaller"`: smaller text.
+
+<LiveExample src='Text.variant.jsx'>
+  <AllExamples.TextVariantExample client:load />
+</LiveExample>
+
```

Comment:
Worth emphasizing the separation of visuals and semantics, and adding a segue to the next section.

```suggestion

Please note that the `variant` prop only affects the visual size and is completely decoupled from the semantics, which can be controlled using the `as` prop (see below).

```

Reply:
I have made the change.